src/app/shell/workshop-details/workshop-page/workshop-page.component.ts

```diff
@@ -13,16 +15,16 @@ interface imgPath {
 @Component({
   selector: 'app-workshop-page',
   templateUrl: './workshop-page.component.html',
-  styleUrls: ['./workshop-page.component.scss']
+  styleUrls: ['./workshop-page.component.scss'],
 })
-
 export class WorkshopPageComponent implements OnInit, OnDestroy {
-  readonly Role: typeof Role = Role;
-  public categoryIcons = CategoryIcons;
   @Input() workshop: Workshop;
   @Input() provider: Provider;
   @Input() providerWorkshops: WorkshopCard[];
   @Input() role: string;
+  readonly Role: typeof Role = Role;
+  public categoryIcons = CategoryIcons;
+  tabIndex: number;
```

Comment:
tabIndex is a little bit confusing naming since it is reserved attribute for tab key navigation

Reply:
what about "activeTabIndex"?

Reply:
why dont you use selectedIndex as an input parameter name

Reply:
replaced with selectedIndex